### PR TITLE
hotfix/Upgrade Dependencies

### PR DESCRIPTION
**Pull request by ggrg**
- Upgraded dependencies
- Fix standard issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,31 +1,8 @@
 {
   "name": "@mojaloop/event-stream-processor",
-  "version": "v8.1.0",
+  "version": "8.3.3-snapshot",
```
Comment (rmothilal) [inline package.json]: Why are you creating a snapshot?

File: package.json [hunk 2]
```diff
@@ -52,22 +29,45 @@
   ],
   "license": "Apache-2.0",
   "bugs": {
-    "url": "https://github.com/mojaloop/email-notifier"
+    "url": "https://github.com/mojaloop/event-stream-processor"
```
Comment (rmothilal) [inline package.json]: Should have /issues at the end of this url

File: package.json (hunk 1 above)
Comment (ggrg) [reply package.json]: Because I've seen the last 3 releases are snapshots. Shall I change it to 8.3.3 and create a normal release?

Comment (vgenev) [reply package.json]: @rmothilal @ggrg  We are still reworking and changed the logger technology before. Also no tests or proper error handling yet. So we shall keep it still as a snapshot, I believe.

**Comment by ggrg** [comment]
I'm closing this then. Thanks, @vgenev.

**Pull request closed without merging**